src/modelize/modelize_property.nit

```diff
@@ -892,7 +892,7 @@ redef class AMethPropdef
 			if param_names.length != msignature.arity then
 				var node: ANode
 				if nsig != null then node = nsig else node = self
-				modelbuilder.error(node, "Redef error: {mpropdef} redefines {mpropdef.mproperty.intro} with {param_names.length} parameter(s), {msignature.arity} expected. Signature is {mpropdef}{msignature}")
+				modelbuilder.error(node, "Redef Error: expected {msignature.arity} parameters like in `{mpropdef.mproperty.intro}`; got {param_names.length}. Signature is `{mpropdef}{msignature}`.")
```

Comment:
I just got this error and I think it could be improved so that useful information can be visible in Vim syntastic.

Old message: `Redef error: fights#Entity#change_action redefines entities#Entity#change_action with 1 parameter(s), 2 expected. Signature is fights#Entity#change_action(turn: GameTurn[Game], new_action: nullable Action)`

Message with this PR: `Redef Error: expected 2 parameter(s) like in entities#Entity#change_action; got 1. Signature is 'fights#Entity#change_action(turn: GameTurn[Game], new_action: nullable Action)'`

My proposed message: `Redef Error: expected 2 parameter(s) for 'change_action(turn: GameTurn[Game], new_action: nullable Action)'; got 1. Introduced in entities#Entity#change_action.`

Notice that I used only the short name of the method (we already have a location).